crates/web-sys/webidls/unstable/scheduling-apis.webidl

```diff
@@ -0,0 +1,58 @@
+enum TaskPriority {
+  "user-blocking",
+  "user-visible",
+  "background"
+};
+
+dictionary SchedulerPostTaskOptions {
+  AbortSignal signal;
+  TaskPriority priority;
+  [EnforceRange] unsigned long long delay = 0;
+};
+
+callback SchedulerPostTaskCallback = any ();
+
+[Exposed=(Window, Worker)]
+interface Scheduler {
+  Promise<any> postTask(SchedulerPostTaskCallback callback,
+                        optional SchedulerPostTaskOptions options = {});
+};
+
+[Exposed=(Window, Worker)]
+interface TaskPriorityChangeEvent : Event {
+  constructor(DOMString type, TaskPriorityChangeEventInit priorityChangeEventInitDict);
+
+  readonly attribute TaskPriority previousPriority;
+};
+
+dictionary TaskPriorityChangeEventInit : EventInit {
+  required TaskPriority previousPriority;
+};
+
+dictionary TaskControllerInit {
+  TaskPriority priority = "user-visible";
+};
+
+[Exposed=(Window,Worker)]
+interface TaskController : AbortController {
+  constructor(optional TaskControllerInit init = {});
+
+  undefined setPriority(TaskPriority priority);
```

Comment:
I probably should have mentioned this _before_ you merged, but I think this needs to be annotated with `[Throws]`. I'm not sure why it isn't in Firefox, but the spec says it's supposed to throw if it gets called reentrantly (i.e. inside a `prioritychange` handler).

Reply:
My bad, should have waited longer, gonna make a fix momentarily.

Reply:
Thanks for catching that!